### ISSUE TITLE
[Antivirus] Bitdefender detects a false positive

### ISSUE THREAD

**Issue by ArthurWolfhound** [markwragg/PowerShell-Lumos#11]
Hi! Just installed Lumos via PSGallery, and on attempt to register scheduled task, my AV became angry (what a surprise Bitdefender, what a surprise)...
Tried to disable literally every switch in settings of my AV looks like it will still protect some core system components nonetheless.

Can you please look at it? (Sorry for not adding it as text - I have no idea how to disable this markdown monstrosity cause it converts powershell output to gibberish...)

![image](https://user-images.githubusercontent.com/16789106/89108143-ca945100-d43e-11ea-8db8-eb8ed7330213.png)
![image](https://user-images.githubusercontent.com/16789106/89108145-cd8f4180-d43e-11ea-9506-939f88f5f8ce.png)

**Comment by markwragg** [comment]
It looks like this is a heuristic based detection: it thinks the act of a script creating a scheduled task seems suspicious so it blocks it. I think your only options are to either temporarily disable your antivirus or to just create the scheduled task manually.

**Comment by ArthurWolfhound** [comment]
Thanks a lot to Bitdefender SRL...

I've actually tried to switch off every possible switch there and it still works, and the only way to actually get rid of him completely is to boot in safe mode.

Found a little crappy workaround: 
1. Add C:\Windows\System32\Tasks to full exceptions
2. Let Lumos register there
3. Change exception to C:\Windows\System32\Tasks\Lumos

Found how to submit false positive: [here](https://www.bitdefender.com/submit/), if you're interested as developer to spend your time of course.

Thanks and good luck.
